test/libsolidity/semanticTests/abiEncoderV2/abi_encode_v2_in_function_inherited_in_v1_contract.sol

```diff
@@ -30,6 +30,9 @@ contract C is B {
 }
 // ----
 // test() -> 77
-// gas irOptimized: 110348
-// gas legacy: 151866
-// gas legacyOptimized: 110373
+// gas irOptimized: 55148
+// gas irOptimized code: 55200
+// gas legacy: 57266
+// gas legacy code: 94600
+// gas legacyOptimized: 55173
+// gas legacyOptimized code: 55200
```

Comment:
I went with this format for simplicity, but I don't like how verbose it is. Comparing gas changes in PR diffs was already tedious even without the new entries.

I think that something along these lines would be more readable, especially when differences are small and when there's highlighting of differences within the line (which seems not enabled in github comments):
```diff
-// gas: 55148+55200 ir (optimized), 7000+94600 legacy, 55173+55200 legacy (optimized)
+// gas: 55148+55200 ir (optimized), 57266+94600 legacy, 55173+55200 legacy (optimized)
```

And even with large differences, the misalignment actually makes it easy to spot such, more important, lines:
```diff
-// gas: 110348 ir (optimized), 151866 legacy, 110373 legacy (optimized)
+// gas: 55148+55200 ir (optimized), 57266+94600 legacy, 55173+55200 legacy (optimized)
```

In the end I did not have time to do that though. I might try later when I have some spare time between bigger tasks.